api/jsonapi.go

```diff
@@ -63,11 +63,14 @@ type jsonAPIHandler struct {
 	node   *core.OpenBazaarNode
 }
 
+var lastManualScan time.Time
+
 func newJSONAPIHandler(node *core.OpenBazaarNode, authCookie http.Cookie, config schema.APIConfig) *jsonAPIHandler {
 	allowedIPs := make(map[string]bool)
 	for _, ip := range config.AllowedIPs {
 		allowedIPs[ip] = true
 	}
+	lastManualScan = time.Now().Add(time.Duration(-10) * time.Minute)
```

Comment:
You don't need to initialize `lastManaualScan`. The zero value would make the check below pass: (https://play.golang.org/p/yHj9vObnYS-) `time.Since(time.Time{}).Minutes() > 10 // true`